docs/querying/sql.md

```diff
@@ -218,9 +218,10 @@ UNION ALL
 SELECT COUNT(*) FROM tbl WHERE my_column = 'value2'
 ```
 
-:::info
- With top-level queries, you can't apply GROUP BY, ORDER BY, or any other operator to the results of a UNION ALL.
-:::
+Keep the following in mind when writing top-level UNION ALL queries:
+
+- You can't apply GROUP BY, ORDER BY, or any other operator to the results of a UNION ALL.
```

Comment:
Can coalesce into a single paragraph as mentioned in the other comment
```suggestion

```

Reply:
This limitation predates my changes. So does it no longer apply to non-MSQ queries?

Reply:
This limitation is still present, but as per the previous comment, we can make it cohesive by coalescing both of the limitations into one big para.

Reply:
Ok, if we want it in 1 paragraph, I'll need to rewrite the paragraph first. Starting with `If you use the MSQ task engine for the query,` sets the reader up to think that the whole paragraph is only about the MSQ task engine limitations

Reply:
https://github.com/apache/druid/pull/15223/commits/9ab413099ce3a7a68f42cd75ba8b08ad29d702c7 has the new paragraph